src/ontology/mondo-edit.obo

```diff
@@ -453438,18 +453367,17 @@ property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-520
 
 [Term]
 id: MONDO:0100343
-name: antenatal Bartter syndrome
-def: "A phenotypic variant of Bartter syndrome presenting antenatally with maternal polyhydramnios, pre-term delivery and postnatally with polyuria, and nephrocalcinosis. Hypokalemic alkalosis, increased levels of plasma renin and aldosterone, low blood pressure and vascular resistance to angiotensin II are characteristically associated. Genotypically they comprise Type 1 and Type 2 Bartter syndrome." [Orphanet:93604]
-comment: Reason of obsoletion: out of scope - MONDO:excludeGroupingClass. This is part of a large mass-obsoletione effort for ORDO terms.
-subset: obsoletion_candidate
+name: obsolete antenatal Bartter syndrome
+def: "OBSOLETE. A phenotypic variant of Bartter syndrome presenting antenatally with maternal polyhydramnios, pre-term delivery and postnatally with polyuria, and nephrocalcinosis. Hypokalemic alkalosis, increased levels of plasma renin and aldosterone, low blood pressure and vascular resistance to angiotensin II are characteristically associated. Genotypically they comprise Type 1 and Type 2 Bartter syndrome." [Orphanet:93604]
 synonym: "Bartter syndrome, furosemide type" EXACT [Orphanet:93604]
 synonym: "Bartter syndrome, furosemide-amiloride type" EXACT [Orphanet:93604]
 synonym: "hyperprostaglandin E syndrome" EXACT [Orphanet:93604]
-xref: Orphanet:93604 {source="MONDO:equivalentObsolete"}
-is_a: MONDO:0015231 {source="Orphanet:93604", source="https://github.com/monarch-initiative/mondo/issues/2935", source="https://orcid.org/0000-0001-5493-2602"} ! Bartter syndrome
+xref: Orphanet:93604 {source="MONDO:obsoleteEquivalentObsolete"}
```

Comment:
@nicolevasilevsky I thought this was `{source="MONDO:obsoleteEquivalent"}` - did this change?